### PR TITLE
Add technologies to README file

### LINKED ISSUE #178: Add  technologies to README file
**I'm submitting a ...**
- [x] feature request

**Current behavior:**
There is no technology section in the Readme

**Expected behavior:**
Readme should have  technologies we are using in the repository and how it work.

**Steps to reproduce:**
- Edit the README file
- Add technology information just like [this](https://github.com/fossasia/meilix-generator#technologies-used).

### PR DESCRIPTION
**Pull request by kvaluruk**
Fixes #178 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added the technology stack into the README file from [here](https://github.com/fossasia/meilix-generator#technologies-used)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,17 @@ It generates badges for every csv file and combines them to one.
...
+### Technologies Used
+
+BadgeYAY uses a number of open source projects to work properly:
+
+* [Flask](http://flask.pocoo.org/) - Microframework powered by python
+* [Bootstrap](http://getbootstrap.com/) - Responsive frontend framework for webapp
+* [Shell](https://en.wikipedia.org/wiki/Unix_shell) - Script used for triggering Travis using [Travis API](https://docs.travis-ci.com/user/triggering-builds/)
+* [Heroku](https://www.heroku.com/) - Webapp deployed here
+* [Travis](travis-ci.org) - Continuous Integration which build the iso
+* [Github Release](https://help.github.com/articles/creating-releases/) - Deploying the iso here
```
Comment (vaibhavsingh97) [inline README.md]: Why GitHub release?? Please understand and  deploy the project locally. There is no iso generated in this project.

File: README.md [hunk 2]
```diff
@@ -62,6 +62,17 @@ It generates badges for every csv file and combines them to one.
...
 
+### Technologies Used
+
+BadgeYAY uses a number of open source projects to work properly:
+
+* [Flask](http://flask.pocoo.org/) - Microframework powered by python
+* [Bootstrap](http://getbootstrap.com/) - Responsive frontend framework for webapp
+* [Shell](https://en.wikipedia.org/wiki/Unix_shell) - Script used for triggering Travis using [Travis API](https://docs.travis-ci.com/user/triggering-builds/)
+* [Heroku](https://www.heroku.com/) - Webapp deployed here
+* [Travis](travis-ci.org) - Continuous Integration which build the iso
```
Comment (vaibhavsingh97) [inline README.md]: Travis is not used for building the iso, it's only check tests

File: README.md [hunk 3]
```diff
@@ -62,6 +62,17 @@ It generates badges for every csv file and combines them to one.
 There is a travis build which build the badges automatically.
 When a PR is merged into the master branch, the current badges can be downloaded.
 
+### Technologies Used
+
+BadgeYAY uses a number of open source projects to work properly:
+
+* [Flask](http://flask.pocoo.org/) - Microframework powered by python
+* [Bootstrap](http://getbootstrap.com/) - Responsive frontend framework for webapp
+* [Shell](https://en.wikipedia.org/wiki/Unix_shell) - Script used for triggering Travis using [Travis API](https://docs.travis-ci.com/user/triggering-builds/)
```
Comment (vaibhavsingh97) [inline README.md]: Shell is not used for triggering Travis build. Please fix this

**Comment by kvaluruk** [comment]
Oh, I misread the issue, my bad. Closing this for now.

**Pull request closed without merging**